lldb/test/API/CMakeLists.txt

```
@@ -74,6 +74,15 @@ else()
   endif()
 endif()
 
+find_program(LLDB_DIRNAME_PATH dirname)
+if(LLDB_DIRNAME_PATH)
+  message(STATUS "Found dirname: ${LLDB_DIRNAME_PATH}")
+else()
+  message(STATUS "Did not find 'dirname'.")
```

```suggestion
  message(STATUS "Could NOT find 'dirname'")
```

This matches what CMake prints for libraries.